website/docs/cdktf/concepts/resources.mdx

```diff
@@ -381,6 +381,240 @@ If you need to use the special [`self` object](/terraform/language/resources/pro
+ts-import-with-configuration  Import without configuration detected. Terraform has created configuration for it:
+                              # __generated__ by Terraform
+                              # Please review these resources and move them into your main configuration files.
+
+                              # __generated__ by Terraform from "best-bucket-in-the-world"
+                              resource "aws_s3_bucket" "bucket" {
+                                bucket              = "best-bucket-in-the-world"
+                                bucket_prefix       = null
+                                force_destroy       = null
+                                object_lock_enabled = false
+                                tags = {
+                                  foo = "bar"
+                                }
+                                tags_all = {
+                                  foo = "bar"
+                                }
+                              }
+
+
+                              CDKTF has translated the code to the following:
+
+                              import { Construct } from "constructs";
+                              /*
+                               * Provider bindings are generated by running `cdktf get`.
+                               * See https://cdk.tf/provider-generation for more details.
+                               */
+                              import { S3Bucket } from "./.gen/providers/aws/s3-bucket";
+                              class MyConvertedCode extends Construct {
+                                constructor(scope: Construct, name: string) {
+                                  super(scope, name);
+                                  new S3Bucket(this, "bucket", {
+                                    bucket: "best-bucket-in-the-world",
+                                    bucketPrefix: [null],
+                                    forceDestroy: [null],
+                                    objectLockEnabled: false,
+                                    tags: {
+                                      foo: "bar",
+                                    },
+                                    tagsAll: {
+                                      foo: "bar",
+                                    },
+                                  });
+                                }
+                              }
+
+
+                              Please review the code and make any necessary changes before adding it to your codebase.
+                              Make sure to only copy the code within the construct's constructor.
+```
+
+Though at this point, your resource(s) has/have not been imported. To import, first add the new generated configuration to your project, then remove the initial call of `generateConfigForImport`. Finally, add the call `importFrom` as it is done in the section "How To Import" above. On apply, your resource(s) will be imported, then becoming managed by CDKTF.
```

Comment:
We could reference the `How to import` section above as it has the details to follow from here on 🤔

Reply:
Shortened what I had a bit and referenced the `How To Import` section more strongly 👍